docs/general-development/custom-security-trimming-for-search-in-sharepoint-server.md

```diff
@@ -86,7 +86,7 @@ To create a custom security post-trimmer for search results, you must create a c
 
 
 
-The **ISecurityTrimmerPost** interface contains two methods that you must implement: [Initialize(NameValueCollection, SearchServiceApplication)](https://msdn.microsoft.com/library/Microsoft.Office.Server.Search.Query.ISecurityTrimmerPost.Initialize.aspx) and **CheckAccess(IList<String>, IList<String>, IDictionary<String, Object>, IIdentity)**.
+The **ISecurityTrimmerPost** interface contains two methods that you must implement: [Initialize(NameValueCollection, SearchServiceApplication)](https://msdn.microsoft.com/library/Microsoft.Office.Server.Search.Query.ISecurityTrimmerPost.Initialize.aspx) and **CheckAccess(IList&lt;String&gt;, IList&lt;String&gt;, IDictionary&lt;String, Object&gt;, IIdentity)**.

```

Comment:
Correct markdown escaping of angled brackets should be prefixed with\; don't change to the HTML encoding.